keyboards/3.0/en-t-k0-teletype.xml

```diff
@@ -28,19 +28,19 @@
     <key id="slash" to="/" />
   </keys>
 
-  <layerMaps form="hardware">
-    <layerMap id="base">
+  <layers form="hardware" hardware="us">
+    <layer id="base">
       <row keys="Q W E R T Y U I O P gap2" />
       <row keys="gap A S D F G H J K L enter gap" />
       <row keys="figs Z X C B V N M ltrs linefeed null" />
```

Comment:
```suggestion
      <row keys="Z X C B V N M ltrs linefeed null" />
```

There is no key left of Z (`figs`) on a `us` base keyboard.

We can't do switch keys (`ltrs`) on a hardware keyboard.

This puts `linefeed` onto the dot key.

What does `null` do?

Reply:
it sends a null… 

Might want to withdraw this particular sample.